package.json

```diff
@@ -108,6 +108,7 @@
     "@angular/forms": "4.0.1",
     "@angular/http": "4.0.1",
     "c3": "0.4.11",
+    "lodash": "^4.17.11",
```

Comment:
why not fixed version?

Reply:
`lodash` was in dev deps, why we need to move here?

Reply:
Because now `lodash` is a runtime dependency., earlier it was only by the dev tools, now it is used in the actual application.

Reply:
I guess `lodash` can stay as devDependency because outs is an Node application, not a Node module/library.

Reply:
> why not fixed version?

This shouldn't matter because I added package-lock.json.